### PR TITLE
[doc] PR for Improve menu titles and other enhancements

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-3878

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/security/ldap-authentication-platform.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ You enable LDAP authentication in the YugabyteDB cluster by setting the LDAP con
 
 This section describes how to configure YugabyteDB Anywhere to use an LDAP server such as Active Directory with TLS.
 
-For more information on LDAP in YugabyteDB, refer to [LDAP Authentication](../../../secure/authentication/ldap-authentication/).
+For more information on LDAP in YugabyteDB, refer to [LDAP authentication](../../../secure/authentication/ldap-authentication/).
 
 ## Bind to the LDAP server using TLS
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/security/ldap-authentication-platform.md:33]: There's a typo in the last sentence "Expect the followingn output"

File: docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md [hunk 2]
```diff
@@ -19,15 +19,15 @@ YugabyteDB Anywhere allows you to protect data in transit by using the following
...
-YugabyteDB Anywhere lets you create a new self-signed certificate, use an existing self-signed certificate, or upload a third-party certificate from external providers, such as Venafi or DigiCert (which is only available for an on-premise cloud provider).
+YugabyteDB Anywhere lets you create a new self-signed certificate, use an existing self-signed certificate, or upload a third-party certificate from external providers, such as Venafi or DigiCert (which is only available for an on-premises cloud provider).
 
-You can enable TLS encryption during universe creation and change these settings for an existing universe.
+You can enable encryption in transit (TLS) during universe creation and change these settings for an existing universe.
 
-## Self-Signed Certificates Generated by YugabyteDB Anywhere
+## Self-signed certificates generated by YugabyteDB Anywhere
 
 YugabyteDB Anywhere can create self-signed certificates for each universe. These certificates may be shared between universes within a single instance of YugabyteDB Anywhere. The certificate name has the following format:
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md:33]: extra comma before parenthesis

File: docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md [hunk 3]
```diff
@@ -102,18 +102,18 @@ When you create a universe, you can enable TLS using your own certificates, as f
 1. Select an existing certificate from the **Root Certificate** list and then select the certificate that you have uploaded.
 1. Create the universe.
 
-You can also modify TLS settings for an existing universe by navigating to **Universes**, opening a specific universe, clicking **Actions > Edit Security > Encryption in-Transit** to open the **TLS Configuration** dialog, and then following the procedure described in [How to Use YugabyteDB Anywhere-Generated Certificates to Enable TLS](#how-to-use-yugabytedb-anywhere-generated-certificates-to-enable-tls) for an existing universe.
+You can also modify TLS settings for an existing universe by navigating to **Universes**, opening a specific universe, clicking **Actions > Edit Security > Encryption in-Transit** to open the **TLS Configuration** dialog, and then following the procedure described in [Use YugabyteDB Anywhere-generated certificates to enable TLS](#how-to-use-yugabytedb-anywhere-generated-certificates-to-enable-tls) for an existing universe.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md:105]: bad link

File: docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md:1]: Long page!

**Review by ddhodge — APPROVED** [review]
Minor fixes

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/_index.md [hunk 5]
```diff
@@ -26,9 +26,20 @@ YugabyteDB Anywhere can create a YugabyteDB universe with many instances (virtua
...
       </div>
       <div class="body">
-        Back up and restore YugabyteDB Anywhere.
+        Back up and restore YugabyteDB Anywhere server.
       </div>
     </a>
   </div>
 
+  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
+    <a class="section-link icon-offset" href="back-up-restore-yp/">
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/administer-yugabyte-platform/_index.md:35]: fixed

File: docs/content/preview/yugabyte-platform/alerts-monitoring/live-queries-dashboard.md [hunk 6]
```diff
@@ -11,34 +11,30 @@ isTocNested: true
...
+All user roles — `Super Admin`, `Admin`, and `Read-only` — are granted permissions to use the **Live Queries** dashboard.
+
+Note that there is no significant performance overhead on databases because the queries are fetched on-demand and are not tracked in the background.
+
+The following table describes the **Live Queries** column values.
+
+| Column                          | Description                                                  |
+| ------------------------------- | ------------------------------------------------------------ |
+| Node Name                       | The YB-TServer node name generated by YugabyteDB Anywhere.   |
+| Private IP                      | The IP address of the database node.                         |
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/alerts-monitoring/live-queries-dashboard.md:29]: Deleted that colon

File: docs/content/preview/yugabyte-platform/alerts-monitoring/live-queries-dashboard.md [hunk 7]
```diff
@@ -11,34 +11,30 @@ isTocNested: true
...
+| Keyspace:                       | The YCQL keyspace or YSQL database used by the query.        |
+| Session status<br/> (YSQL only) | The YSQL session status: idle, active, idle in transaction, fastpath function call, idle in transaction (aborted), or disabled. |
+| Query                           | The query command. <br>Example:<br>`select * from my_keyspace.my_table` |
+| Elapsed time                    | The duration (in milliseconds) of the query handling.        |
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/alerts-monitoring/live-queries-dashboard.md:33]: Fixed

File: docs/content/preview/yugabyte-platform/manage-deployments/upgrade-software.md [hunk 8]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Upgrade the YugabyteDB software
 headerTitle: Upgrade the YugabyteDB software
-linkTitle: Upgrade YugabyteDB software
+linkTitle: Upgrade YugabyteDB
 description: Use YugabyteDB Anywhere to upgrade the YugabyteDB software.
 menu:
   preview:
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/manage-deployments/upgrade-software.md:7]: Fixed

File: docs/content/preview/yugabyte-platform/security/ldap-authentication-platform.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/security/ldap-authentication-platform.md:33]: fixed

File: docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md (hunk 3 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md:105]: Fixed

File: docs/content/preview/yugabyte-platform/alerts-monitoring/live-queries-dashboard.md [hunk 9]
```diff
@@ -11,34 +11,30 @@ isTocNested: true
...
+| Type<br>(YCQL only)             | Shows the YCQL query type: PREPARE, EXECUTE, QUERY, or BATCH |
+| Client host                     | The address of the client that sent this query.              |
+| Client port                     | The port of the client that sent this query.                 |
+| Client name                     | The client name used to execute the query.                   |
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/alerts-monitoring/live-queries-dashboard.md:37]: Fixed

**Pull request merged**